stackdriver/README.md

```diff
@@ -0,0 +1,61 @@
+# stackdriver
+
+This gem is a convenience package for loading all Stackdriver gems in the google-cloud-ruby project. 
```

Comment:
"This gem instruments a Ruby web application for Stackdriver diagnostics. When loaded, it integrates with Rails, Sinatra, or other Rack-based web frameworks to collect application diagnostic and monitoring information for your application.

Specifically, this gem is a convenience package that loads and automatically activates the instrumentation features of the following gems:"